app/helpers/ai_helper.rb

```diff
@@ -98,6 +157,7 @@ def ai_create_workout(user_query)
         exercise.station = Station.all.sample
```

Comment:
Assigning a random station to each exercise?

Reply:
think this was done earlier just to satisfy the foreign key for seeding at the time, but michael has an update with more advance seeds , so i imagine this random station will be removed in future when his code is merged.